public/css/index.css

```diff
@@ -0,0 +1,118 @@
+:root {
+  /* colors */
+  --main-bg-color: #1e1e25;
+  --form-background-color: #27293d;
+  --form-input-main-border-color: #2a324e;
+  --form-input-focus-color: #fb00ff;
+  --form-input-focus-bg-color: #1d253b;
+  --button-main-color: gray;
+  --button-blue-color: #305cf4;
+  --button-pink-color: #d54fd7;
+  /* Borders */
+  --basic-border-size: 1px;
+  --form-border-radius: 12px;
+  --form-input-radius: 16px;
+  --button-border-radius: 10px;
```

Comment:
może to uspójnić i dać 1 max 2 bordery?
```css
--border-sm: 10px
--border-md: 16px
```
border 1px jest niewidoczny praktycznie